doc/templates/documentation.html

```diff
@@ -0,0 +1,113 @@
+{% extends "layout.html" %}
+{% set title = 'Documentation scikit-learn: machine learning in Python' %}
+{%- block extrahead %}
+<script>
+    VERSION_SUBDIR = (function(groups) {
+        return groups ? groups[1] : null;
+    })(location.href.match(/^https?:\/\/scikit-learn.org\/([^\/]+)/));
+</script>
+{% endblock %}
+{% block content %}
+<div class="container sk-landing-container body" role="main">
+  <h1 class="sk-documentation-index-header mb-3">Documentation of scikit-learn {{ version }}</h1>
+  <div class="row no-gutters">
+    <div class="col-md-4 mb-2">
+      <div class="card h-100 sk-documentation-index-card mx-md-2 mb-2">
+        <a href="tutorial/basic/tutorial.html" class="stretched-link sk-documentation-index-anchor"></a><h4 class="card-header">Quick Start</h4>
+        <div class="card-body p-3">
+          <p class="card-text">A very short introduction into machine learning problems and how to solve them using scikit-learn. Presents basic concepts and conventions.</p>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-4 mb-2">
+      <div class="card h-100 sk-documentation-index-card mx-md-2 mb-2">
+        <a href="user_guide.html" class="stretched-link sk-documentation-index-anchor"></a><h4 class="card-header">User Guide</h4>
+        <div class="card-body p-3">
+          <p class="card-text">The main documentation. This contains an in-depth description of all algorithms and how to apply them.</p>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-4 mb-2">
+      <div class="card h-100 sk-documentation-index-card mx-md-2 mb-2">
+        <h4 class="card-header">Other Versions</h4>
+        <div class="card-body p-3">
+          <p class="card-text">
```

Comment:
Don't wrap a `<ul>` in a `<p>`. The browser will automatically close the `<p>` and so insert some visual padding north of the list.

(Perhaps do some HTML validation)

Reply:
I'll do more HTML validation on more pages to make sure everything is okay.